wagtail/images/wagtail_hooks.py

```diff
@@ -95,7 +94,7 @@ def register_image_feature(features):
                 'id': True,
             }
         }, js=[
-            versioned_static('wagtailimages/js/image-chooser-modal.js'),
+            'wagtailimages/js/image-chooser-modal.js',
```

Comment:
As far as I can see, the `versioned_static` call that's been removed here never gets added back anywhere else, so we lose the cache-busting behaviour - `wagtail.admin.rich_text.editors.draftail.features.Feature` wraps this in a `Media` object with no modifications, and that object is then used to build `DraftailRichTextArea.plugin_media` and ultimately the result of `DraftailRichTextArea.media`. Reinstating the `versioned_static` will presumably involve refactoring `DraftailRichTextArea` similarly to `HalloRichTextArea`, so that building the media happens lazily in the `media` property rather than on `__init__`. (As an aside, we should probably use [`@cached_property`](https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.functional.cached_property) on both classes so that the evaluation only happens once.)